README.md

```diff
@@ -896,6 +896,7 @@ commcell_password  |   no  |  | |  Commcell password |
 client  |   yes  |  | |  The name of the Client. | 
 backupset  |   no  |  default backupset  | |  The name of the backupset. | 
 subclient  |   no  |  subclient named default.  | |  The name of the subclient. | 
+backup_level  |  no  |  Incremental  |  Full Incremental Differential Synthetic_full  |  Backup Level.  |
```

Comment:
Add these options using a unordered list (use commvault.ansible.request for reference)

Reply:
I'm confused.  Do you mean in the README.md file?  Do you mean like this?

**backup_level  |  no  |  Incremental  |  ["Full,"Incremental", "Differential", "Synthetic_full"]  |  Backup Level.  |**

Reply:
Yes in the README.md file, the choices are defined in an unordered list
Have a look at the documentation for commvault.ansible.request in README.md

```
commcell_password  |   no  |  | |  Password | 
method  |   yes  |  | <ul> <li>GET</li>  <li>POST</li>  <li>PUT</li>  <li>DELETE</li> </ul> |  Web method to use to make the request | 
url  |   yes  |  | |  The web URL or service to run the HTTP request on. Use "{0}" to autopopulate webconsole URL | 
```